### DISCUSSION TITLE
`php_server` failing due to permissions?

### DISCUSSION THREAD

**Discussion by splashsky** [php/frankenphp#851] [Q&A] [answered]
I've been using FrankenPHP for a while now and things have been great! I usually install Caddy through apt then copy the Caddy service file over and just swap Caddy for FrankenPHP. This has been flawless for me since I began using it. I am using Debian through WSL2, and after doing an update that updated my PHP binaries and other packages, Franken now has this odd bug where when I'm using the `php_server` directive, I get this error in Edge:
![image](https://github.com/dunglas/frankenphp/assets/12104206/f9b2cf34-2be3-4a3c-9f78-210849861a37)
But this does not happen when not using the `php_server` directive.

**Comment by AlliBalliBaba** [comment]
Are you using basic authentication in your application? If the page works in Chrome but not in Edge that could be the issue, since Edge does not properly support basic authentication yet.

**Reply by splashsky** [reply, marked as answer]
No I was not. Seems that the caddy user was unable to access the files I had been serving all of a sudden, which was especially odd because I had already been using the same config for months. Weird little hiccups. I fixed it by running the FrankenPHP process as my user.

**Reply by withinboredom** [reply]
> since Edge does not properly support basic authentication yet.

I've been using basic auth + edge for over a year now, what do you mean?
